platform/defaults.go

```diff
@@ -52,6 +52,16 @@ const (
 // via a credential helper, or via the `CNB_REGISTRY_AUTH` environment variable. See [auth.DefaultKeychain] for further information.
 const EnvUseDaemon = "CNB_USE_DAEMON"
 
+// ## Provided to handle inputs and outputs in OCI layout format
+
+// The lifecycle can be configured to read the inputs images like `run-image` or `previous-image` in OCI layout format instead of a
```

Comment:
```suggestion
// The lifecycle can be configured to read the input images like `run-image` or `previous-image` in OCI layout format instead of from a
```